backend/src/main/java/net/es/oscars/resv/svc/ConnService.java

```
@@ -1142,8 +1142,14 @@ public Pair<SimpleConnection, Connection> holdConnection(SimpleConnection in) th
         Connection c = simpleToHeldConnection(in);
         prettyLog(c);
 
-        this.held.put(connectionId, c);
+        Validity v = this.validate(in, ConnectionMode.NEW);
+        in.setValidity(v);
 
+        if (!v.isValid()) {
+            log.info("{} : not valid because {}", in.getConnectionId(), v.getMessage());
```

If it's not a valid connection, should we throw an error and return?